### PR TITLE
Update route approach (#2546)

### LINKED ISSUE #2546: Multiple approach to setting routes of APIs in the Bit Platform projects
Currently, there are some different approach to setting APIs route in Bit Platform projects that needs to be fixed in a way that only one unique approach is used everywhere.

### PR DESCRIPTION
**Pull request by saleh-ghezelbash**
This closes #2546

### CODE REVIEW

**Review by msynk — APPROVED** [review]

**Review by msynk — CHANGES_REQUESTED** [review]
it seems there are still other controllers that require these improvements

File: src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Pages/Components/DataGrid/BitDataGridDemo.razor.cs [hunk 1]
```diff
@@ -735,7 +735,7 @@ protected override async Task OnInitializedAsync()
                 query.Add(""$orderby"", string.Join("", "", req.GetSortByProperties().Select(p => $""{p.PropertyName} {(p.Direction == BitDataGridSortDirection.Ascending ? ""asc"" : ""desc"")}"")));
             }
 
-            var url = NavManager.GetUriWithQueryParameters(""Products/GetProducts"", query);
+            var url = NavManager.GetUriWithQueryParameters(""/api/Products/GetProducts"", query);
```
Comment (ysmoradi) [inline src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Pages/Components/DataGrid/BitDataGridDemo.razor.cs:738]: It was working without starting / before
Do we need this now?

**Review by ysmoradi — APPROVED** [review]

File: src/Sales/Bit.Sales.WebSite/Api/Controllers/ContactUsController.cs [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 namespace Bit.Sales.WebSite.Api.Controllers;
 
-[Route("api/[controller]")]
+[Route("api/[controller]/[action]")]
```
Comment (msynk) [inline src/Sales/Bit.Sales.WebSite/Api/Controllers/ContactUsController.cs:8]: are we sure about this change?
@ysmoradi

File: src/Templates/TodoTemplate/Bit.TodoTemplate/Api/Controllers/TodoItemController.cs [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 namespace TodoTemplate.Api.Controllers;
 
-[Route("api/[controller]")]
+[Route("api/[controller]/[action]")]
```
Comment (msynk) [inline src/Templates/TodoTemplate/Bit.TodoTemplate/Api/Controllers/TodoItemController.cs:6]: again, are we sure about this one too?
@ysmoradi

**Review by msynk — CHANGES_REQUESTED** [review]

**Review by ysmoradi — APPROVED** [review]

**Review by msynk — APPROVED** [review]

**Pull request merged**